apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js

```diff
@@ -12,8 +12,12 @@ export default function OnboardingCard({
     subtitle,
     completed,
 }) {
+    const [isActive, setIsActive] = useState(false);
+    const detectHover = event => {
+        setIsActive(current => !current);
+      };
     return (
-        <OnboardingCardContainer completed = {completed}>
+        <OnboardingCardContainer completed = {completed} className={isActive ? 'hovered' : ''} onMouseEnter={detectHover} onMouseLeave={detectHover}>
```

Comment:
For a hover effect, you don't need to actually create the functionality to know whether someone has hovered over an html element/tag. There is a CSS selector you can use to easily do this. have a look at this to see how: https://www.w3schools.com/cssref/sel_hover.php

Reply:
Reverted.